{
  "contract": "0x26e7ddfd8cd4e4c6e1272b0ab53f14a1b0d6170c",
  "tool": "mythril",
  "start": 1563338086.2223656,
  "end": 1563338108.5362878,
  "duration": 22.31392216682434,
  "analysis": {
    "error": null,
    "issues": [
      {
        "address": 3425,
        "code": " >= a",
        "debug": "calldatasize_UmbrellaCoin: 0x1000000000000000000000000000000000000000000000000000000000000044\nstorage_keccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncalldata_UmbrellaCoin_4: 0xf1592bedbbff7fd743ed18b69e49c6a2d5afe5b9c6df6bd8868438480103db39\ncalldata_UmbrellaCoin_32 + 4: 0x6173538839847ecb1c051d3958c52c0d15261637901c038184c1b94b3d5bd9a5\nstorage_keccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x6173538839847ecb1c051d3958c52c0d15261637901c038184c1b94b3d5bd9a5\ncalldata_UmbrellaCoin_0: 0xa9059cbb00000000000000000000000000000000000000000000000000000000\ncallvalue: 0x0\n",
        "description": "A possible integer overflow exists in the function `transfer(address,uint256)`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.",
        "filename": "/unique_contracts/0x26e7ddfd8cd4e4c6e1272b0ab53f14a1b0d6170c.sol",
        "function": "transfer(address,uint256)",
        "lineno": 21,
        "title": "Integer Overflow ",
        "type": "Warning"
      },
      {
        "address": 3438,
        "code": "return c;\n  }\n",
        "debug": "The exception is triggered under the following conditions:\n\ncalldatasize_UmbrellaCoin: 0x1000000000000000000000000000000000000000000048\nstorage_keccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncalldata_UmbrellaCoin_4: 0x9000000000000c0000004020000000001b03002097fffecbfc01fe3fd37ff57c\ncalldata_UmbrellaCoin_32 + 4: 0x980000000000a3fffffebfbffffffeffe0dfff85a080003bc66c000000000000\nstorage_keccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x980000000000a3fffffebfbffffffeffe0dfff85a080003bc66c000000000000\ncalldata_UmbrellaCoin_0: 0xa9059cbb00000000000000000000000000000000000000000000000000000000\ncallvalue: 0x0\n",
        "description": "A reachable exception (opcode 0xfe) has been detected. This can be caused by type errors, division by zero, out-of-bounds array access, or assert violations. This is acceptable in most situations. Note however that `assert()` should only be used to check invariants. Use `require()` for regular input checking. ",
        "filename": "/unique_contracts/0x26e7ddfd8cd4e4c6e1272b0ab53f14a1b0d6170c.sol",
        "function": "transfer(address,uint256)",
        "lineno": 22,
        "title": "Exception state",
        "type": "Informational"
      },
      {
        "address": 3462,
        "code": "\n    return a ",
        "debug": "The exception is triggered under the following conditions:\n\ncalldatasize_UmbrellaCoin: 0x48\nstorage_keccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x0\ncalldata_UmbrellaCoin_32 + 4: 0x1\ncalldata_UmbrellaCoin_0: 0xa9059cbb00000000000000000000000000000000000000000000000000000000\ncallvalue: 0x0\n",
        "description": "A reachable exception (opcode 0xfe) has been detected. This can be caused by type errors, division by zero, out-of-bounds array access, or assert violations. This is acceptable in most situations. Note however that `assert()` should only be used to check invariants. Use `require()` for regular input checking. ",
        "filename": "/unique_contracts/0x26e7ddfd8cd4e4c6e1272b0ab53f14a1b0d6170c.sol",
        "function": "transfer(address,uint256)",
        "lineno": 16,
        "title": "Exception state",
        "type": "Informational"
      }
    ],
    "success": true
  }
}